### PR TITLE
Add confirmation to various actions (#324)

### LINKED ISSUE #324: Enhancement: Add action confirmation
Add action confirmation for «Reset to defaults» in « Media uploader». Right now, there is no confirmation of the action, so if you accidentally missclick, you will have to fill everything out again.

Also add action confirmation for «Clear uploads» in «Recent uploads».

And also when we delete a channel through «Manage channels» confirmation is required, and if we delete it through  ⋮ → channel → remove channel, then it is not there.

### PR DESCRIPTION
**Pull request by rod41732**
Fixes #324 

Add confirmation dialogs when:
- deleting channels through the overflow menu ( `Overflow Menu -> Channel -> Remove Channel)
- resettings image uploader settings
- clearing upload

### CODE REVIEW

File: app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt [hunk 1]
```diff
@@ -17,9 +17,8 @@ import android.widget.ProgressBar
 import android.widget.TextView
 import androidx.activity.OnBackPressedCallback
 import androidx.activity.result.PickVisualMediaRequest
-import androidx.activity.result.contract.ActivityResultContracts
-import androidx.activity.result.contract.ActivityResultContracts.*
-import androidx.activity.result.launch
+import androidx.activity.result.contract.ActivityResultContracts.PickVisualMedia
+import androidx.activity.result.contract.ActivityResultContracts.StartActivityForResult
```
Comment (flex3r) [inline app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt:21]: Unrelated changes

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -118,7 +119,7 @@
     <string name="uploader_headers">Headers</string>
     <string name="uploader_image_link">Media link pattern</string>
     <string name="uploader_deletion_link">Deletion link pattern</string>
-    <string name="uploader_description">You can set a custom host for uploading media, like imgur.com or s-ul.eu. DankChat uses the same configuration format as Chatterino.\nCheck this guide for help: https://wiki.chatterino.com/Image%20Uploader/</string>
+    <string name="uploader_description">You can set a custom host for uploading media, like imgur.com or s-ul.eu. DankChat uses the same configuration format as Chatterino.\nCheck this guide for help: https://wiki.chatterino.com/image%20Uploader/</string>
```
Comment (flex3r) [inline app/src/main/res/values/strings.xml:122]: What is this change?

**Review by flex3r — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt (hunk 1 above)
Comment (rod41732) [reply app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt:21]: seems Organizing Imports in Android Studio cause this change, FeelsDankMan

Comment (rod41732) [reply app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt:21]: should I revert tho?

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (rod41732) [reply app/src/main/res/values/strings.xml]: editing the file and accidentally changed, reverted, FeelsDankMan.

File: app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt (hunk 1 above)
Comment (flex3r) [reply app/src/main/kotlin/com/flxrs/dankchat/main/MainFragment.kt:21]: Weird, oh well

**Review by flex3r — APPROVED** [review]
LGTM image
